SokolTextGame/Commands.cs

```diff
@@ -110,6 +129,33 @@ public void Execute(string[] words, World world)
             }
         }
     }
+    public class Talk : ICommand
+    {
+        public void Execute(string[] words, World world)
+        {
+            Console.ForegroundColor = ConsoleColor.Red;
+            if (string.Join(" ", words) == "talk") Console.Write("blah blah blah blah\nThe command exists, but you have to say \"talk to\"\n");
+            else if (string.Join(" ", words) == "talk to")
+            {
+                if (string.IsNullOrEmpty(string.Join(" ", world.CurrentLocation().ObjectOnLocation)))
```

Comment:
Please remove this entire nested if. This leads to inconsistent behavior: `talk to guard` should produce `There is no such thing here` when used anywhere but the square. Instead, it produces `There are no interlocutors at the moment`, which makes it unclear and confusing.
So you should only have `if words == "talk to" then write(Talk to who?)` and that's it.